lib/stream/make_bundle.js

```diff
@@ -0,0 +1,140 @@
+var _ = require("lodash");
+var through = require("through2");
+var hasES6 = require("../graph/has_es6");
+var unbundle = require("../graph/unbundle");
+var flattenBundle = require("../bundle/flatten");
+var makeBundle = require("../bundle/make_bundle");
+var splitByBuildType = require("../bundle/split_by_build_type");
+var addTraceurRuntime = require("../bundle/add_traceur_runtime");
+var makeBundlesConfig = require("../bundle/make_bundles_config");
+
+var BUNDLE_DEPTH = 1;
+
+module.exports = function() {
+	return through.obj(function(data, enc, done) {
+		try {
+			var result = bundle(data);
+			done(null, result);
+		} catch(err) {
+			done(err);
+		}
+	});
+};
+
+function makeAddNpmPackagesNode(npmContext) {
```

Comment:
I think this code (and the code that calls it) should go in lib/bundle in its own module. We have other stuff like this, like [add_steal](https://github.com/stealjs/steal-tools/blob/master/lib/bundle/add_steal.js). This should follow that convention where it gets a bundle and unshifts itself into the top.